### PR TITLE
Profile reskin to match Deck and Notebook

### PR DESCRIPTION
**Pull request by bfollington**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/1062

After much deliberation, I decided I actually do kinda like the full gradient experience on the profile.

I can see the argument for leaving the header with a solid colour background though (see the comment below).

Simulator Screenshot - iPhone 15 Pro - 2024-01-24 at 17 13 07Simulator Screenshot - iPhone 15 Pro - 2024-01-24 at 17 12 49

### CODE REVIEW

**Comment by bfollington** [comment]
The more conservative version:

simulator_screenshot_A5D1FC00-FDB1-4D46-845B-8B278803C6E4simulator_screenshot_1B77329F-7DA4-4E24-9890-7744265D4CDC

File: xcode/Subconscious/Shared/Components/Common/EllipsisLabelView.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct EllipsisLabelView: View {
     var body: some View {
         Image(systemName: "ellipsis")
-            .frame(width: AppTheme.minTouchSize + AppTheme.padding, height: AppTheme.minTouchSize + AppTheme.unit2)
+            .frame(width: AppTheme.minTouchSize, height: AppTheme.minTouchSize)
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/Common/EllipsisLabelView.swift:14]: Deliberate?

**Review by gordonbrander — APPROVED** [review]
LGTM

File: xcode/Subconscious/Shared/Components/Common/EllipsisLabelView.swift (hunk 1 above)
Comment (bfollington) [reply xcode/Subconscious/Shared/Components/Common/EllipsisLabelView.swift:14]: yep, I changed the layout of these cards

**Pull request merged**